tests/TestCase.php

```diff
@@ -2,40 +2,17 @@
 
 namespace Tipoff\Authorization\Tests;
 
-use Illuminate\Database\Eloquent\Factories\Factory;
-use Orchestra\Testbench\TestCase as Orchestra;
 use Tipoff\Authorization\AuthorizationServiceProvider;
+use Tipoff\Support\SupportServiceProvider;
+use Tipoff\TestSupport\BaseTestCase;
 
-class TestCase extends Orchestra
+class TestCase extends BaseTestCase
 {
-    public function setUp(): void
-    {
-        parent::setUp();
-
-        Factory::guessFactoryNamesUsing(
-            fn (string $modelName) => 'Tipoff\\Authorization\\Database\\Factories\\'.class_basename($modelName).'Factory'
-        );
-    }
-
     protected function getPackageProviders($app)
     {
         return [
+            SupportServiceProvider::class,
             AuthorizationServiceProvider::class,
```

Comment:
Including `AuthorizationServiceProvider::class` in the list of package providers will be necessary to create the permissions tables.